tests/utils/test_metric_utils.py

```diff
@@ -0,0 +1,61 @@
+"""Unit tests for metrics utilities.
+
+Author: Travis Driver
+"""
+
+import unittest
+
+import trimesh
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler, Cal3Bundler
+from gtsfm.common.keypoints import Keypoints
+
+import gtsfm.utils.metrics as metric_utils
+
+
+class TestMetricUtils(unittest.TestCase):
+    """Class containing all unit tests for metric utils."""
+
+    def test_mesh_inlier_correspondences(self):
```

Comment:
nit: would be nice to add a return type here for consistency
```python
def test_mesh_inlier_correspondences(self) -> None:
```

Reply:
fixed

Reply:
marking as resolved, thanks